packages/terra-date-picker/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ Unreleased
 
 4.5.0 - (July 2, 2019)
 ------------------
+### Added
+* Aria-Label added to input to fix accesssibility issue of Date-Time-Picker.
```

Comment:
Move this log to the already existing "Added" section.

Reply:
Resolved in 009e0a3

Reply:
Looks like this got moved to an already released section again. Happens whenever you merge master in if there was a release.